### PR TITLE
ARROW-8960: [MINOR] [FORMAT] fix typo

### PR DESCRIPTION
**Pull request by dota17**

### CODE REVIEW

File: cpp/src/arrow/vendored/string_view.hpp [hunk 1]
```diff
@@ -986,7 +986,7 @@ nssv_constexpr bool operator>= (
 
 #if ! nssv_CPP11_OR_GREATER || nssv_BETWEEN( nssv_COMPILER_MSVC_VERSION, 100, 141 )
 
-// accomodate for older compilers:
+// accommodate for older compilers:
```
Comment (wesm) [inline cpp/src/arrow/vendored/string_view.hpp:989]: Please revert. This is vendored code

File: cpp/src/arrow/vendored/uriparser/UriBase.h [hunk 2]
```diff
@@ -179,7 +179,7 @@ typedef struct UriIp6Struct {
 } UriIp6; /**< @copydoc UriIp6Struct */
 
 
-struct UriMemoryManagerStruct;  /* foward declaration to break loop */
+struct UriMemoryManagerStruct;  /* forward declaration to break loop */
```
Comment (wesm) [inline cpp/src/arrow/vendored/uriparser/UriBase.h:182]: Please revert. Vendored code

File: CHANGELOG.md [hunk 3]
```diff
@@ -50,7 +50,7 @@
 * ARROW-7631 - [C++][Gandiva] return zero if there is an overflow while converting a decimal to a lower precision/scale
 * ARROW-7680 - [C++][Dataset] Partition discovery is not working with windows path
 * ARROW-7701 - [C++] [CI] Flight test error on macOS
-* ARROW-7713 - [Java] TastLeak was put at the wrong location
+* ARROW-7713 - [Java] TasteLeak was put at the wrong location
```
Comment (wesm) [inline CHANGELOG.md:53]: Please revert changes in this file, it is generated from JIRA, so if you want to fix the typos, you need to fix the JIRA issue titles

File: LICENSE.txt [hunk 4]
```diff
@@ -1785,7 +1785,7 @@ preceding version 3 has the following license:
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  *
- * The licence and distribution terms for any publically available version or
+ * The licence and distribution terms for any publicly available version or
```
Comment (wesm) [inline LICENSE.txt:1788]: Please revert. We want to preserve the text of the original code license

File: cpp/CHANGELOG_PARQUET.md [hunk 5]
```diff
@@ -339,7 +339,7 @@ Parquet C++ 1.0.0
     * [PARQUET-626] - Fix builds due to unavailable llvm.org apt mirror
     * [PARQUET-629] - RowGroupSerializer should only close itself once
     * [PARQUET-472] - Clean up InputStream ownership semantics in ColumnReader
-    * [PARQUET-739] - Rle-decoding uses static buffer that is shared accross threads
+    * [PARQUET-739] - Rle-decoding uses static buffer that is shared across threads
```
Comment (wesm) [inline cpp/CHANGELOG_PARQUET.md:342]: Please revert

Comment (fredgan) [reply cpp/CHANGELOG_PARQUET.md]: All done! Thanks

File: CHANGELOG.md [hunk 6]
```diff
@@ -866,7 +866,7 @@
 * ARROW-5809 - [Rust] Dockerize (add to docker-compose) Rust Travis CI build
 * ARROW-5831 - [Release] Migrate and improve binary release verification script
 * ARROW-5855 - [Python] Add support for Duration type
-* ARROW-5859 - [Python] Support ExtentionType on conversion to numpy/pandas
+* ARROW-5859 - [Python] Support ExtensionType on conversion to numpy/pandas
```
Comment (kiszk) [inline CHANGELOG.md:869]: According to https://github.com/apache/arrow/pull/7274#discussion_r430541033, it looks necessary to revert all of changes in `CHANGELOG.md`.

Comment (fredgan) [reply CHANGELOG.md]: OK, Thanks, now all the modification in `CHANGELOD.md` have been reverted.

**Review by wesm — APPROVED** [review]
+1

**Pull request closed without merging**